example/src/App.tsx

```diff
@@ -66,6 +66,10 @@ const App = () => {
     const [isNativeUIMRecShowing, setIsNativeUIMRecShowing] = useState(false);
     const [isNativeAdShowing, setIsNativeAdShowing] = useState(false);
     const [statusText, setStatusText] = useState('Initializing SDK...');
+    const [preloadedBannerId, setPreloadedBannerId] = useState<AdViewId>();
+    const [preloadedMRECId, setPreloadedMRECId] = useState<AdViewId>();
```

Comment:
`MREC` is used for user-facing strings. In code, we'd do `preloadedMRecId, setPreloadedMRecId`.